### ISSUE TITLE
Weird black overlays and missing textures over people after moving up/down a z-level/appearance is altered while on multi-z maps

### ISSUE THREAD

**Issue by necromanceranne** [tgstation/tgstation#82517]
Reporting client version: 515.1630

## Issue Summary

![image](https://github.com/tgstation/tgstation/assets/40847847/20770e6a-0712-46bc-8d29-f18fbfb87a70)

Bit hard to see here, but this guy is actually in a fully activated engineering modsuit. No alterations. It simply disappeared and was replaced with a black plane over his appearance. This seems to also happen whenever an appearance change is called.

Any time I'd shift between z-levels, my own character and many others would end up having this really weird visual bug. This seems to also happen whenever an appearance change is called. Say, I shifted my gas mask out of the way or put on a new piece of clothing.

## Round ID:
[226767](https://scrubby.melonmesa.com/round/226767)

## Testmerges:
- [Cleans up the SS13_base lua file and adds a new lua file for easily handling multiple signals on different objects.](https://www.github.com/tgstation/tgstation/pull/82458)
- [General maintenance for chem master](https://www.github.com/tgstation/tgstation/pull/82002)
- [Plane Master Refactor + Mutliz Rendering Optimizations (Bigly)](https://www.github.com/tgstation/tgstation/pull/82419)

## Reproduction:

Basically, walk down a flight of stairs. There isn't much too it, it happens very consistently. The only solution to fix it seems to be reconnecting.

**Comment by MrMelbert** [comment]
Assuming this is #82419

**Comment by necromanceranne** [comment]
I was just about to come back and post that myself

**Comment by LemonInTheDark** [comment]
It's possible this is a client version issue I think there may have been a bugfix I'm relying on here or something.
What are your multiz render preferences?

**Comment by necromanceranne** [comment]
Standard

**Comment by LemonInTheDark** [comment]
hmmm

**Comment by SyncIt21** [comment]
@MrMelbert test merge PR is no longer open

**Comment by necromanceranne** [comment]
This likely is no longer necessary as the testmerge has finished
